### PR TITLE
RATEST-63 Write allergy page

### PR DESCRIPTION
**Pull request by achilep**
Issue I on: https://issues.openmrs.org/browse/RATEST-63

### CODE REVIEW

**Comment by HerbertYiga** [comment]
A better description (RATEST-63: Write allergy page

**Comment by HerbertYiga** [comment]
cc @k-joseph

File: src/test/java/org/openmrs/contrib/qaframework/page/AllergyPage.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.uitestframework.page.Page;
```
Comment (kaweesi) [inline src/test/java/org/openmrs/contrib/qaframework/page/AllergyPage.java:3]: to my plan, addOrEdit should be the same page as this, the other second is the allergies which rather should include allergy list etc

File: src/test/java/org/openmrs/contrib/qaframework/page/AllergyPage.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebElement;
+
+public class AllergyPage extends Page {
```
Comment (kaweesi) [inline src/test/java/org/openmrs/contrib/qaframework/page/AllergyPage.java:8]: once you have sorted out my previous feedback, please add all possible functionality onto the respective page. thanks

**Comment by kaweesi** [comment]
Closed this up. this page is now existing in master

**Pull request closed without merging**